src/utils/filterContests.test.js

```diff
@@ -0,0 +1,18 @@
+import { filterContests } from './filterContests';
+
+function testFilter(contests, from, to) {
+  let filteredContests = contests.filter((contest) => {
+    return contest.startTime >= from && contest.startTime <= to;
+  });
+  contests = filterContests(contests, from, to);
+  return expect(contests).toMatchObject(filteredContests);
+}
+
+test('filter contests', () => {
+  let cs = [];
+  for(let i = 1; i <= 10; i++)
+    cs.push({
+      startTime: new Date(i * 1000)
```

Comment:
I actually know a very good pattern for doing stuff like this. It's called test Data Builder. Take a look: http://blog.codeleak.pl/2014/06/test-data-builders-and-object-mother.html

The idea is to centralize the data generated using another module. Although this test makes sense, you are not testing a Contest type, just a regular object with the desired property. A Contest type must have a lot of other properties. I can help you code the builder.

With the builder, we can easily make better real contests for testing, not just 1000*i starting date objects.